### PR TITLE
Prefer calloc over malloc

### PR DESCRIPTION
**Pull request by AtariDreams**
malloc + memset can be better expressed as calloc.

### CODE REVIEW

File: tcpslice.c [hunk 1]
```diff
@@ -891,13 +890,11 @@ extract_slice(struct state *states, const int numfiles, const char *write_file_n
 	last_state = 0;
 	last_hdr.ts.tv_sec = last_hdr.ts.tv_usec = 0;
 	last_hdr.caplen = last_hdr.len = 0;
-	last_pkt = (u_char*) malloc(snaplen);
+	last_pkt = (u_char*) calloc(snaplen, sizeof(u_char));
```
Comment (infrastation) [inline tcpslice.c:893]: `calloc(1, ...` would be more consistent with the code style.

Comment (AtariDreams) [reply tcpslice.c:893]: We are doing the multiplication anyway in calloc. It's best to use calloc as intended.

**Comment by AtariDreams** [comment]
@infrastation Addressed!

**Comment by infrastation** [comment]
Thank you.

**Pull request merged**